### DISCUSSION TITLE
PC Always Offline

### DISCUSSION THREAD

**Discussion by jmcgarry** [AlexGustafsson/homebridge-wol#249] [Questions] [answered]
Hi, 

Very new to homebridge and home automation but I wanted to test out WOL. After a lot of trial and error I finally got it to work but I'm running into an issue where Homekit doesn't see my PC as online. It will WOL but always sees it as offline. Any ideas why this maybe? 

https://ibb.co/nLt7bxD

**Comment by thebaz** [comment]
Is your PC a Windows PC? If so, did you enable the rule for ping in the firewall?

**Comment by Niklas31** [comment, marked as answer]
https://superuser.com/questions/1683853/cannot-ping-a-windows-11-machine
